frontend/src/visualization/filtering/ShortestPathMenu.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { Box, Button } from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
+import ShortestPathNodeSelection from './ShortestPathNodeSelection';
+
+/**
+ * Menu where start and end node for shortest path can be selected
+ */
+export default function ShortestPathMenu(): JSX.Element {
+  const [errorNotSpecified, setErrorNotSpecified] = useState(false);
+
+  const [startNode, setStartNode] = useState<number | null>(null);
+  const [endNode, setEndNode] = useState<number | null>(null);
+
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Why is this needed? This is not an empty function. Is this an artifacts that is remaining from the development process, where this was empty?

Reply:
yes its an artificat, I will delete that